setup.py

```diff
@@ -12,43 +9,40 @@
     ensure_targets,
     get_version,
 )
-
-from setuptools import setup
+import setuptools
 
 
-log.set_verbosity(log.DEBUG)
-log.info('setup.py entered')
-log.info('$PATH=%s' % os.environ['PATH'])
+# due to https://github.com/jupyterlab/jupyterlab/blob/136d2ec216ebfc429a696e6ee75fee5f8ead73e2/jupyterlab/federated_labextensions.py#L347
+# we should not print out anything, otherwise setup.py --name gives noise
+# log.set_verbosity(log.DEBUG)
+# log.info('setup.py entered')
+# log.info('$PATH=%s' % os.environ['PATH'])
 
 LONG_DESCRIPTION = 'A Python/ThreeJS bridge utilizing the Jupyter widget infrastructure.'
 
-here = os.path.abspath(os.path.dirname(sys.argv[0]))
+HERE = Path(__file__).parent.resolve()
 name = 'pythreejs'
-version = get_version(os.path.join(here, name, '_version.py'))
+lab_path = (HERE / name / "labextension")
 
+version = get_version(HERE / name / '_version.py')
 
 cmdclass = create_cmdclass(
     'js',
     data_files_spec=[
-        ('share/jupyter/nbextensions/jupyter-threejs',
-         name + '/static',
-         '*.js'),
-        ('share/jupyter/nbextensions/jupyter-threejs',
-         name + '/static',
-         '*.js.map'),
```

Comment:
Should we no longer be shipping the nbextension?

Reply:
I'm fed up with making this mistake, good catch!